package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hackerapi-frontend",
   "homepage": "https://app.mchacks.ca",
-  "version": "0.1.0",
+  "version": "1.4.0",
```

Comment:
what's up with the version bump?

Reply:
it wasn't updated before, just matching the latest release https://github.com/hackmcgill/hackerAPI-frontend/releases